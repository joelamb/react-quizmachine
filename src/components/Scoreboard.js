import React from 'react';
import '../styles/scoreboard.scss'

const Scoreboard = ({ score, lives }) => (
  <section className="scoreboard">
    <p className="scoreboard__lives">Lives<span>{lives}</span></p>
    <p className="scoreboard__score">Score<span>{score}</span></p>
  </section>
)

export default Scoreboard;