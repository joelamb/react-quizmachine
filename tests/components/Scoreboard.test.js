import React from 'react';
import Scoreboard from '../../src/components/Scoreboard';
import renderer from 'react-test-renderer';

describe('Scoreboard', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Scoreboard />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})