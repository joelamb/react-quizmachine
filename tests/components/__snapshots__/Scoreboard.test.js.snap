// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scoreboard renders correctly 1`] = `
<section
  className="scoreboard"
>
  <p
    className="scoreboard__lives"
  >
    Lives
    <span />
  </p>
  <p
    className="scoreboard__score"
  >
    Score
    <span />
  </p>
</section>
`;
